### PR TITLE
Lynbarry's first Goodie

### PR DESCRIPTION
**Pull request by Lynbarry**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
It says that I'm awesome! ;)

**What problem does your Instant Answer solve (Why is it better than organic links)?**

**What is the data source for your Instant Answer? (Provide a link if possible)**

**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

**Which existing Instant Answers will this one supercede/overlap with?**

**Are you having any problems? Do you need our help with anything?**

**Where did you hear about DuckDuckHack? (For first time contributors)**

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screenshot from 2015-03-01 17 59 37

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/Lynbarry.pm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+category "special";
+topics "special_interest", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/Lynbarry.pm";
+attribution github => ["https://github.com/Lynbarry", "Lynbarry"],
+            twitter => "quetzco";
+
+triggers start => "duckduckhack lynbarry";
+
+handle remainder => sub {
+        return if $_;
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/Lynbarry.pm]: Can you fix the indentation here? It should be 4 spaces :)

**Comment by mintsoft** [comment]
@Lynbarry Congrats on your first goodie, there's just the very minor indentation thing to fix then we can get it merged!

File: lib/DDG/Goodie/IsAwesome/Lynbarry.pm (hunk 1 above)
Comment (Lynbarry) [inline lib/DDG/Goodie/IsAwesome/Lynbarry.pm]: I messed up with git and had to do a merge, but I think the indentation is fixed now.

**Comment by mintsoft** [comment]
@Lynbarry Excellent, thanks for the quick response. Congratuations on your first Goodie! :tada:

**Comment by zekiel** [comment]
@Lynbarry  out of curiosity, where did you find out about DuckDuckHack?

**Comment by moollaza** [comment]
@Lynbarry congrats! It's live: https://duckduckgo.com/?q=duckduckhack+Lynbarry&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Comment by Lynbarry** [comment]
@zekiel I think I read about it on the hiring-page... ;)

@moollaza Nice!!

**Comment by zekiel** [comment]
thanks!

**Pull request merged**